packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -74,9 +74,11 @@
     "Specifies whether the commands for change-set-based development are available to the user.",
   "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor",
   "force_source_retrieve_text": "SFDX: Retrieve Source from Org",
-  "force_source_retrieve_this_source_file_text":
-    "SFDX: Retrieve This Source File from Org",
```

Comment:
@ruthemmanuelle Previously, if a user right clicked inside of an open file in the text editor context, the command text for deploy and retrieve were "SFDX: Retrieve This Source File from Org" and "SFDX: Deploy This Source File to Org". I realized that this isn't completely accurate in the case where a user right clicks on an open manifest file, since it is not just the manifest file but also all of the metadata entities specified in the manifest that get deployed or retrieved. To account for that, I removed the word `File` from both strings. Does that sound ok? 
<img width="680" alt="screen shot 2018-08-28 at 1 38 10 pm" src="https://user-images.githubusercontent.com/1639375/44746452-a21fb180-aac7-11e8-814e-a1d8a87321bb.png">

Reply:
Yes, this is good. Sorry I missed your question before.